README.md

```diff
@@ -238,20 +238,16 @@ and go into it with `docker exec -it kukkuu-backend bash` first.
      - Rebuild the container with `docker compose up --build` to take the package changes into account
 
 ### Code format
-
-This project uses [Ruff](https://docs.astral.sh/ruff/) for code formatting and quality checking.
-
-Basic `ruff` commands:
-
-* lint: `ruff check`
-* apply safe lint fixes: `ruff check --fix`
-* check formatting: `ruff format --check`
-* format: `ruff format`
-
 [`pre-commit`](https://pre-commit.com/) can be used to install and
 run all the formatting tools as git hooks automatically before a
 commit.
 
+This project uses [Ruff](https://docs.astral.sh/ruff/) for code formatting and quality checking through pre-commit.
+
+Available `ruff` commands configured to pre-commit:
+
+* apply safe lint fixes: `pre-commit run ruff`
+* format: `pre-commit run ruff-format`
```

Comment:
`pre-commit run -a` is probably the easiest way to run these, although it will run the other checks as well.

Reply:
oh yes the -a flag I'll add that 👍 
(But I'll preserve the specific `ruff `and `ruff-format` since that's what the section is about)